### ISSUE TITLE
PySide6: False positives with signals

### ISSUE THREAD

**Issue by bersbersbers** [python/mypy#13833]
**Bug Report**

(A clear and concise description of what the bug is.)

**To Reproduce**

```python
"""Bug"""
from PySide6.QtCore import QFileSystemWatcher

w = QFileSystemWatcher()
w.fileChanged.connect(None)
print("Done.")
```

**Expected Behavior**

mypy passes

**Actual Behavior**

> bug.py:5: error: "QFileSystemWatcher" has no attribute "fileChanged"
Found 1 error in 1 file (checked 1 source file)

**Your Environment**

- Mypy version used: mypy 0.982 (compiled: yes)
- Mypy command-line flags: none
- Mypy configuration options from `mypy.ini` (and other config files): none
- Python version used: 3.10.7

Similar behavior with 
- QCheckBox.stateChanged
- QSpinBox.valueChanged
- QDoubleSpinBox.valueChanged
- QSlider.valueChanged
- QComboBox.currentIndexChanged
- QRadioButton.toggled
- QGroupBox.toggled
- QCheckBox.toggled
and a bunch more.

**Comment by bersbersbers** [comment]
I have found https://bugreports.qt.io/browse/PYSIDE-1318 but it seems unrelated - that bug reports missing properties for signals. In this issue, mypy reports missing signals.

**Comment by JelleZijlstra** [comment]
This does look like a bug on PySide6's side. It appears they ship their own stub files, which mypy would use to understand the library. If the stubs are incomplete, mypy will report errors.

**Comment by bersbersbers** [comment]
Thanks - reported at https://bugreports.qt.io/browse/PYSIDE-2073

**Comment by bersbersbers** [comment]
Duplicate of https://bugreports.qt.io/browse/PYSIDE-1603
